### PR TITLE
Ts config debugger

### PR DESCRIPTION
**Pull request by wzalazar**
#314 

- [x] Configuration vscode, server
- [x] Docker files for development
- [x] Docker compose for development
- [x] New commands in package.json for development
- [x] Nodemon
- [x] Task to compile in makefile
- [x] Startup easy with make development, adding the task compile in this command

### CODE REVIEW

File: node/src/blockchain/connection.ts [hunk 1]
```diff
@@ -25,11 +25,19 @@ const defaultConfiguration: ConnectionConfiguration = {
   }
 }
 
+const isDevelopment = () => {
+  return process.env.NODE_ENV === 'development';
```
Comment (lautarodragan) [inline node/src/blockchain/connection.ts]: Avoid semicolons

File: node/src/blockchain/connection.ts [hunk 2]
```diff
@@ -25,11 +25,19 @@ const defaultConfiguration: ConnectionConfiguration = {
...
+}
+
+const getExtension = () => {
+  return isDevelopment() ? 'js' : 'ts';
```
Comment (lautarodragan) [inline node/src/blockchain/connection.ts]: Avoid semicolons

File: node/src/blockchain/connection.ts (hunk 1 above)
Comment (wzalazar) [reply node/src/blockchain/connection.ts]: done

File: node/src/blockchain/connection.ts (hunk 2 above)
Comment (wzalazar) [reply node/src/blockchain/connection.ts]: done

**Pull request merged**